<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSignalEditor</class>
 <widget class="QDialog" name="DialogSignalEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="treeView"/>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Filter</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="2">
             <widget class="QLineEdit" name="filter_RTR"/>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="filter_ID_select">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>SID only</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EID only</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SID and EID</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="filter_DLC"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>RTR</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>ID mask</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QLineEdit" name="filter_ID"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>DLC</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>EID/SID</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="3">
             <widget class="QLineEdit" name="filter_Data"/>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Data mask</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Display</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QComboBox" name="disp_Type">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Script</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>INT16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UINT16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>INT32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UINT32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>INT64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UINT64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hex</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Float</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Display mask</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="disp_btn_edit">
              <property name="text">
               <string>Edit script</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Unit</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="disp_Mask"/>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="disp_Unit"/>
            </item>
           </layout>
           <zorder>label_6</zorder>
           <zorder>disp_Type</zorder>
           <zorder>label_7</zorder>
           <zorder>disp_btn_edit</zorder>
           <zorder>label_15</zorder>
           <zorder>disp_Mask</zorder>
           <zorder>disp_Unit</zorder>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Graph</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Scale</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="graph_Scale"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="graph_Color"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Test</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>RTR</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="test_DLC"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="test_RTR">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="test_ID"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>DLC</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Output</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QLineEdit" name="test_Input"/>
            </item>
            <item row="5" column="0" colspan="3">
             <widget class="QLineEdit" name="test_Output"/>
            </item>
            <item row="2" column="0" colspan="3">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Input</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Open|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
